### PR TITLE
[RDAPI-30068] Update Aug22 and May23 with info on XalanJ removal and …

### PR DESCRIPTION
**Pull request by JohnD-Axway**
…restoration

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link
Aug22
http://pr-2670.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20220830_apimgr_relnotes/#apache-xalanj-removed
http://pr-2670.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20220830_apimgr_relnotes/#fixed-security-vulnerabilities

May23
http://pr-2670.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20230530_apimgr_relnotes/#apache-xalanj-reinstated

I'll update the domain when built)

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | b161310 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2670.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md [hunk 1]
```diff
@@ -120,6 +120,12 @@ Setting the cache refresh interval of a LDAP authentication via Auth Repository
 
 API Gateway 7.7 and API Manager 7.7 support OpenJDK JRE. This update includes Zulu OpenJDK v8u345.
 
+### Apache XalanJ removed
+
+CVE-2022-34169 was found in apache xalanJ prior to release, with no fix available. To mitigate this issue the API Gateway was updated to use a version of Xalan shipped with Zulu JRE that has been patched for this CVE.
+To use the patched version of Xalan in scripts and filters it is necessary to replace 'org.apache.xalan.xsltc.trax.TransformerFactoryImpl' with 'com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl'.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md:126]: (replace ‘org.apache.xalan.xsltc.trax.TransformerFactoryImpl’ with ‘com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl’) --- where, in the jvm.xml file?

**Review by mmilner-axway — APPROVED** [review]
Looks good to me in both release notes. Workaround in Aug22 also applies to Nov22 and Feb23.

**Comment by mmilner-axway** [comment]
Yes I agree we don't duplicate the pages and customer should know to check every release note in between the current and target version.
It's not a system property we have introduced to perform a functional change but it has always been in there to remap the classes so it doesn't belong in '[System property changes](http://master.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_reference/system_props/)'. Also it's system/conf/jvm.xml where it is defined already and not conf/jvm.xml or instance conf/jvm.xml.  So I think we should be changing "your system/conf/jvm.xml" not "your jvm.xml" just to avid confusion.

**Comment by JohnD-Axway** [comment]
> So I think we should be changing "your system/conf/jvm.xml" not "your jvm.xml" just to avoid confusion.

I made this change! Thanks

**Comment by andreamussap** [comment]
Changes are published in the production site now - https://docs.axway.com/bundle/axway-open-docs/page/docs/apim_relnotes/20220830_apimgr_relnotes/index.html#apache-xalanj-is-removed

**Pull request merged**